docs/source/tutorials/configure_huggingface_optimizations.md

```diff
@@ -0,0 +1,50 @@
+(Configuring-Huggingface-Optimizations)=
+
+# Configuring Huggingface Optimizations
+
+
+## Introduction
+This document describes how to configure a optimization workflow for Huggingface models, where the user can specify the
+1. Model name: is in Huggingface model hub, e.g. `bert-base-uncased`.
+2. Task name: introduces the task specific head for the model, e.g. `text-classification`. More task names can be found [here](https://huggingface.co/tasks). Task name is used to:
+    - Load model in which user can just provide the model name and task name. Olive will automatically load the model from Huggingface model hub for specific task.
+    - Load specific tokenizer and data processor. Currently, we simplify the user experience only for `text-classification` task, which will be demonstrated in the following examples. *For other tasks, it is under development actively.*
+3. The task specific dataset: is hosted in [Huggingface datasets](https://huggingface.co/datasets), e.g. `glue` dataset for text classification task.
+4. Metric name: is supported by [Huggingface evaluate](https://huggingface.co/docs/evaluate/index). User can refer to [huggingface metrics](https://huggingface.co/metrics) for full metrics list.
+
+## Example
+
+### Model config and dataset config
+Take `bert-base-uncased` as an example, user can specify task name as `text-classification` and dataset as `glue` to form the Huggingface config `hf_config` as follows:
+```json
+"hf_config": {
+    "model_name": "bert-base-uncased",
+    "task": "text-classification",
+    "dataset": {
+        "data_name":"glue",
+        "subset": "mrpc",
+        "split": "validation",
+        "input_cols": ["sentence1", "sentence2"],
+        "label_cols": ["label"],
+        "batch_size": 1
+    }
+}
+```
+Please refer to [hf_config](../overview/options.md#hf_config) for more details.
+
+### Metric config
+```json
+{
+    "name": "accuracy",
+    "type": "accuracy",
+    "backend": "huggingface_metrics",
+    "sub_types": [
+        {"name": "accuracy", "priority": -1},
+        {"name": "f1"}
+    ]
+}
+```
+Please refer to [metrics](../overview/options.md#metrics) for more details.
+
+### Full example
+For the complete example, please refer to [Bert Optimization](https://github.com/microsoft/Olive/tree/main/examples/bert#bert-optimization).
```

Comment:
There are many BERT configs under BERT example. Better to point to the exact one with HF settings, e.g. bert_ptq_cpu.json. But even bert_ptq_cpu.json doesn't have HF metric part.

Reply:
Add a new hf_bert_ptq_cpu.json to demo the huggingface workflow.